### ISSUE TITLE
Adding Syntax section

### ISSUE THREAD

**Issue by cshaa** [OscarGodson/HTML6#24]
Here, in the HTML6 documentation, there is nearly no desc. of the syntax - only a reference to HTML5 and the Section 4 - Tag types. But full documentation has to include it.
Here's a list of syntax articles:
- [ ] what is a tag, what are attributes
  - tag in layman's terms
  - tag types [PARTIALLY]
  - attributes in layman's terms
  - normative description of tag
  - normative description of attributes
  - regular expersion
- [ ] namespaces (+how to include)
- [ ] how to parse wrong code

At this point we don't know anything about namespaces. Will there be something like xmlns? Or will they be included in doctype like this: `<!DOCTYPE html,form>`? And what languages to use to define new namespaces?

**Comment by OscarGodson** [comment]
Only the two you mentioned, and yes, we for sure should have a syntax. section.

**Comment by OscarGodson** [comment]
And, I don't think we need to include it in the DOCTYPE. The point of having namespaces is that you, as a developer, don't have to worry about future namespaces. Is there a benefit you see in having them manually listed out?

**Comment by philandy** [comment]
Could we make the DOCTYPE an attribute of the html tag?

**Comment by cshaa** [comment]
@OscarGodson Listing namespaces is just a XML habit. It's useful for browser - to know what API to load. Eg `<!DOCTYPE html,svg>` would mean load "html" and "svg" APIs. If you wouldn't list it, browser would treat svg tags as ordinary semantic tags and therefore would not render the picture.
@philandy Yep, thats also a solution.

**Comment by philandy** [comment]
@m93a, Oscar's HTML6 is not XHTML; we should be able to assume default behavior for certain tags beyond the namespaces.
